setup.py

```diff
@@ -30,6 +30,7 @@
     },
 
     install_requires=[
-        'pylint>=1.4.0',
+        'pylint==1.4.4',
+        'pylint-django==0.6.1',
```

Comment:
The pattern that I thought was the most well-established was:
* `requirements.txt` lists dependencies to use the software
* `setup.py` reads `requirements.txt` and passes the contents to the `install_requires` block
* If the software has additional dependencies for development, such as for running tests or building documentation, those dependencies are either listed in the `tests_require` section of `setup.py`, or listed in a secondary requirements file (such as `dev-requirements.txt`) which is similarly read into `setup.py`

Is there some documentation that we can refer to, to determine the correct pattern? I'm willing to be proven wrong.